com.unity.render-pipelines.universal/Runtime/Passes/DrawObjectsPass.cs

```diff
@@ -125,9 +123,6 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
                     : new Vector4(flipSign, 0.0f, 1.0f, 1.0f);
                 cmd.SetGlobalVector(ShaderPropertyId.scaleBiasRt, scaleBias);
 
-                context.ExecuteCommandBuffer(cmd);
-                cmd.Clear();
```

Comment:
Same here, I believe this is needed because the [RenderingUtils.RenderObjectsWithError ](https://github.com/Unity-Technologies/Graphics/pull/6877/files#diff-e992d4c5e40f17f7ef44b9f3b4522526758d52ea042a50b5de6bc36aa42deac8L162)calls draw renderers in the context.